src/js/components/container/LogosStripe.jsx

```diff
@@ -22,10 +22,10 @@ class LogosStripe extends RestSyncingComponent {
         localLogos.push(<img src={this.state.data.local[i].image} />)
       }
       return [
-        <div id='global-logos'>{globalLogos}</div>,
-        <div id='local-logos'>
+        <div id='global-logos' className={this.props.showLogos ? '' : 'logos-hidden'}>{globalLogos}</div>,
```

Comment:
The class name should represent the meaning of the change in css. The global logos aren't hidden, and so adding a `logos-hidden` flag to their element is confusing. This should be `full height`, or go ahead and just removed all of the conditions here, and just hider it in css using the one class in the `row`

Reply:
I can't generify this, as both of them have different conditions for both the things that we change when the setting is changed

Reply:
Notice in `App.jsx` how we pass the `showLogos` into `RankingsTable`. You can instead just add a class to the fist `div` there, with the `className=app...`, and that way you can use the CSS to show/hide any elements under it using, for instance, `.show-logos #rankings...`. This is one way to reduce the amount of places we use this condition.

Another option is inside the `logosStripe` file, add the class to `logos-stripe-row` and the use the CSS in the same way. 

The third option I see is to just change the class here to show that the `global-logos` are not hidden. because an element `<global-logos className='logos-hidden'>` makes it look like the logos inside the global-logos stripe are hidden.